saleor/product/management/commands/populatedb.py

```diff
@@ -26,6 +26,15 @@ def handle(self, *args, **options):
             self.stdout.write(msg)
 
         if options['createsuperuser']:
-            user = User.objects.create_superuser(
-                email='admin@example.com', password='admin')
-            self.stdout.write('Superuser - %s' % user.email)
+            credentials = {'email': 'admin@example.com', 'password': 'admin'}
+            user, created = User.objects.get_or_create(
+                email=credentials['email'],
+                is_active=True, is_staff=True, is_superuser=True)
```

Comment:
Shouldn't these be moved to `defaults`? What if the user with that email exists but is not active or is not staff?